pytest test case failed

I try to run the project locally and get failed while run `pytest`
here's the error log
```
python -m pytest
============================================================================================= test session starts ==============================================================================================
platform darwin -- Python 3.12.4, pytest-8.3.3, pluggy-1.5.0
rootdir: datafusion-ray
configfile: pyproject.toml
collected 1 item                                                                                                                                                                                               

datafusion_ray/tests/test_context.py F                                                                                                                                                                   [100%]

=================================================================================================== FAILURES ===================================================================================================
_____________________________________________________________________________________________________ test _____________________________________________________________________________________________________

    def test():
>       ctx = Context(1, False)
E       TypeError: Context.__new__() takes 1 positional arguments but 2 were given

datafusion_ray/tests/test_context.py:22: TypeError
=========================================================================================== short test summary info ============================================================================================
FAILED datafusion_ray/tests/test_context.py::test - TypeError: Context.__new__() takes 1 positional arguments but 2 were given
============================================================================================== 1 failed in 0.82s ===============================================================================================
```

--- a/README.md
+++ b/README.md
@@ -102,7 +102,7 @@ Whenever rust code changes (your changes or via `git pull`):
 
 ```bash
 # make sure you activate the venv using "source venv/bin/activate" first
-maturin develop python -m pytest 
+maturin develop; python -m pytest 
 ```
 
 ## Testing

--- a/datafusion_ray/tests/test_context.py
+++ b/datafusion_ray/tests/test_context.py
@@ -19,6 +19,6 @@ import pytest
 from datafusion_ray import Context
 
 def test():
-    ctx = Context(1, False)
+    ctx = Context(1)
     ctx.register_csv('tips', 'examples/tips.csv', True)
     ctx.plan("SELECT * FROM tips")

